<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>696</width>
    <height>529</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnAdd">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>130</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>为UAT添加内网能力</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnMod">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>180</y>
      <width>221</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>收回能力</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDel">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>370</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>打开网络设置</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnUAT">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>130</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>切换为UAT网络</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWork">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>210</y>
      <width>201</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>切换为内网网络</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>300</y>
      <width>271</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>请将内网网卡更名为：以太网</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnFix">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>240</y>
      <width>431</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>解析内网可访问但UAT下无法访问的网站（暂无效）</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnTest">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>420</y>
      <width>131</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>测试！危险！</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>696</width>
     <height>30</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
